newIDE/app/src/ProjectManager/index.js

```diff
@@ -368,6 +385,14 @@ export default class ProjectManager extends React.Component<Props, State> {
     this._onProjectItemModified();
   };
 
+  _onOpenLayoutProperties = (layout: gdLayout) => {
+    this.setState({ lastOpenLayout: layout, layoutPropertiesDialogOpen: true });
```

Comment:
You don't need both lastOpenLayout and layoutPropertiesDialogOpen.
Just use a single variable: `editedPropertiesLayout`, which is either `null`(nothing is edited) or a `gdLayout` (in which case, the dialog is opened with the layout).

Reply:
Done.